gormschema/gorm.go

```diff
@@ -187,3 +224,52 @@ func WithJoinTable(model any, field string, jointable any) Option {
 		})
 	}
 }
+
+func indirect(t reflect.Type) reflect.Type {
+	for t.Kind() == reflect.Ptr {
+		t = t.Elem()
+	}
+	return t
+}
+
+type (
+	// ViewOption configures a viewBuilder.
+	ViewOption  func(*viewBuilder)
+	viewDefiner interface {
+		ViewDef(driver string) []ViewOption
+	}
```

Comment:
Let expose this interface and comment for the driver enums
```suggestion
	ViewDefiner interface {
		ViewDef(driver string) []ViewOption
	}
```